types/page.ts

```diff
@@ -223,15 +223,15 @@ export const undergraduateAdmission: SegmentNode = {
 };
 
 export const earlyAdmission: SegmentNode = {
-  name: '수시',
+  name: '수시 모집',
   segment: 'early-admission',
   isPage: true,
   parent: undergraduateAdmission,
   children: null,
 };
 
 export const regularAdmission: SegmentNode = {
```

Comment:
그냥 아래 대학원이랑 변수명 형식 통일을 위해
```suggestion
export const undergraduateRegularAdmission: SegmentNode = {
```
이렇게 바꿔줘도 될듯! 귀찮으면 그냥 냅둬도 됩니다,,,🤣

Reply:
수시모집 정시모집 다 바꿨습니다ㅏ